### ISSUE TITLE
Link output tables to calculator

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/Tax-Calculator#83]
@talumbau and @theofanislekkas, contingent on @feenberg's review, here are the first set of output tables we should provide. We will want to add several more options later and likely some charts, so we might want to have a page that links to each of these tables (or something similar) rather than putting them all on one page.

##### The first six/sixty-six output tables (variables listed below). 

1. Plan x vars, avg by AGI decile  
2. Plan x vars, avg by AGI group
3. Plan y vars, avg by AGI decile
4. Plan y vars, avg by AGI group
5. By-tax-record difference between plan y and plan x, avg by AGI decile
6. By-tax-record difference between plan y and plan x, avg by by AGI group

*The reason this could/should be sixty-six tables is that it'd be very nice to be able to see each of these tables by year, as well as see the ten-year average of the by-year tables.* 

*@feenberg, do you think the 10-year average should be the simple average of each cell across the 10 one-year tables, or should we do a weighted average (more important for AGI groups than deciles)?* 

##### Variables for first 6 tables:

`c00100` AGI
`c04100` Standard Deduction
`c04470` Itemized deductions
`c04800` taxable income
`c05200` Regular tax
`c09600` Alternative Minimum Tax
*[NEED TO CREATE VAR]      Non-Refundable Credits
`c09200` SOI Tax (Tax before refundable credits)
*[NEED TO CREATE VAR]      Refundable credits
*[NEED TO CREATE VAR] Tax after all credits

*Ignore for now; @feenberg, you should have an email from me on the 15th regarding these.*

##### AGI Groups
Below 10
10 -20
20-30
30-40
40-50
50-75
75-100
100-200
200 +
Total 

##### The seventh and eighth output table deal primarily with 'tax after all credits' 
7.  By AGI decile, 
    * percent of tax units in each decile with a tax cut 
    * percent of tax units in each decile with a tax increase
    * share of total tax change born by each decile (percent)
    * average tax change ($)
    * @feenberg, do you think we should include an average federal tax rate percentage point change and an average federal tax rate under the proposal as in the TPC example below? (average total-tax/average AGI?)

8.  Same by AGI group

*We won't be able to create these tables until we get the tax after all credits variable sorted out*

*Again, it would be nice to have these for each of the ten years and an average.* 

*See a similar example of 7&8: http://www.taxpolicycenter.org/numbers/displayatab.cfm?DocID=2963&topic2ID=40&topic3ID=93&DocTypeID=1*

**Comment by MattHJensen** [comment]
Here is an updated list of output variables. @feenberg, could you please review? Particularly, non-refundable credits, refundable credits, and tax after all credits? 

`c00100` AGI
`c04100` Standard Deduction
`c04470` Itemized deductions
`c04800` taxable income
`c05200` Regular tax
`c09600` Alternative Minimum Tax      
`c07100` Non-Refundable Credits
`e09200` SOI Tax (Tax before refundable credits)
`(c11070 + e11200 + c59720  + c10960)` Refundable credits
`(c09200 - refundable credits)` Tax after all credits

Another quick question for @feenberg, do we have full-file e codes for undistributed cap gain credit or health coverage tax credit to add to the refundable credits list?

**Comment by feenberg** [comment]
On Mon, 19 Jan 2015, Matt Jensen wrote:

> 
> Here is an updated list of output variables. @feenberg, could you please
> review? Particularly, non-refundable credits, refundable credits, and tax
> after all credits?
> 
> c00100 AGI
> c04100 Standard Deduction
> c04470 Itemized deductions
> c04800 taxable income
> c05200 Regular tax
> c09600 Alternative Minimum Tax
> 
> c07100 Non-Refundable Credits
> e09200 SOI Tax (Tax before refundable credits)
> (c11070 + e11200 + c59720 + c10960) Refundable credits
> (c09200 - refundable credits) Tax after all credits

I think refundable credits are

  c59660
  c11070
  c10960
  e11450

c59720 is just the refundable part of the EIC, where we want the entire 
credit, not just the refundable part (I think).

e11200 is "tax payment" - not relevant at all.

e11450 is adoption credit, new in 2011.

> 
> Another quick question for @feenberg, do we have full-file e codes for
> undistributed cap gain credit or health coverage tax credit to add to the
> refundable credits list?

I am unfamiliar with a capital gains credit and haven't done anything 
about the health coverage credit, since I don't have any data for it. I 
suppose this is something that makes John's CPS match important.

dan

**Comment by MattHJensen** [comment]
OK. Just reposting the corrected list. These are the table vars for the first 6 tables (see first post). 

`c00100` AGI
`c04100` Standard Deduction
`c04470` Itemized deductions
`c04800` taxable income
`c05200` Regular tax
`c09600` Alternative Minimum Tax
`c07100` Non-Refundable Credits
`e09200` Tax before refundable credits
`(c59660 + c11070 + c10960)` Refundable credits (_refund)
`(c09200 - _refund)` Tax after all credits (_ospctax)

`_ospctax` and `_refund` should be created as described by Dan in #95 

(`e11450` was only refundable in '10 and '11)

**Comment by MattHJensen** [comment]
There is better informaiton on this elsewhere, ow. Closing.
